staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/generated.pb.go

```diff
@@ -213,63 +216,77 @@ func (m *OwnerReference) Reset()                    { *m = OwnerReference{} }
 func (*OwnerReference) ProtoMessage()               {}
 func (*OwnerReference) Descriptor() ([]byte, []int) { return fileDescriptorGenerated, []int{27} }
 
+func (m *PartialObjectMetadata) Reset()                    { *m = PartialObjectMetadata{} }
```

Comment:
you'll need to rebase on master... this got picked up and fixed in https://github.com/kubernetes/kubernetes/pull/77403